### PR TITLE
[ADD] Add maturity levels Document

### PR DESCRIPTION
**Pull request by elicoidal**
cc @OCA/board

### CODE REVIEW

**Comment by elicoidal** [comment]
can we merge this? @OCA/board

File: website/Contribution/oca_maturity_level_development_status.rst [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+
+-  Must have at least one declared maintainer.
+
+-  Must depend only on Mature modules.
+
+-  Must not have relevant overlap with existing Mature modules (unless a
+   good justification is given, evaluated by the relevant PSC).
+
+   .. note::
+      The canonical URL to this document is https://odoo-community.org/page/maturity-levels
```
[the remark this answers is not in the document]
Comment (sbidoul) [reply website/Contribution/oca_maturity_level_development_status.rst]: @elicoidal it's https://odoo-community.org/page/development-status

**Comment by sbidoul** [comment]
@elicoidal can you change the document name to `oca_module_lifecycle_development_status.rst` to align with the maintainer role document?

**Comment by max3903** [comment]
Superseeded by #26

**Comment by elicoidal** [comment]
@max3903 maybe you went a little bit too quickly (canonical URL): I will fix it directly in the repo.

**Comment by max3903** [comment]
@elicoidal I needed the link to work for my presentation. 

How do we get 
https://odoo-community.org/page/development-status
to redirect to 
https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/oca_module_lifecycle_development_status.rst?

It is still pointing to the Google Doc... Thanks!

**Comment by sbidoul** [comment]
@max3903 I adapted the redirection on odoo-community.org. Should be ok now.

**Comment by max3903** [comment]
@sbidoul Out of curiosity, how do you do that?

**Comment by sbidoul** [comment]
In the nginx config because I've nothing better that I know of in our Odoo instance.

**Pull request merged**